packages/dd-trace/src/format.js

```diff
@@ -91,7 +89,6 @@ function extractTags (trace, span) {
     addTag(trace.meta, trace.metrics, 'language', 'javascript')
   }
 
-  addTag(trace.meta, trace.metrics, SAMPLING_PRIORITY_KEY, priority)
   addTag(trace.meta, trace.metrics, ORIGIN_KEY, origin)
```

Comment:
Do we need the same change here as well on origin? only set it on the local root span?

> You must set the _dd.origin tag on the local root trace with the value from the X-Datadog-Origin header.

Reply:
I'll revert this change instead as we don't have a good way to handle the concept of a local root right now and it would require pretty much a rewrite of the exporter.